<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetAdopterWindow</class>
 <widget class="QMainWindow" name="PetAdopterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pet Adopter Login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout">
      <item row="0">
       <widget class="QLabel" name="labelFirstName">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditFirstName"/>
      </item>
      <item row="1">
       <widget class="QLabel" name="labelLastName">
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditLastName"/>
      </item>
      <item row="2">
       <widget class="QLabel" name="labelUsername">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditUsername"/>
      </item>
      <item row="3">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditPassword"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer>
        <property name="orientation">
            <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
        </property>
    </spacer>
    <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
            <string>Cancel</string>
        </property>
    </widget>
</item>
<item>
    <spacer>
        <property name="orientation">
            <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
        </property>
    </spacer>
    <widget class="QPushButton" name="pushButtonSubmit">
        <property name="text">
            <string>Submit</string>
        </property>
        <property name="maximumWidth">40</property> <!-- Adjust the maximum width -->
    </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
